errorstyle.css

```diff
@@ -0,0 +1,66 @@
+/* nunito-regular - latin */
+body {
```

Comment:
Most of this CSS looks the same as existing CSS on the site. integrating with jekyll will allow this site-wide css to be automatically applied.